Sources/ChatManager.swift

```diff
@@ -401,16 +401,14 @@ func pathFriendlyVersion(of component: String) -> String {
 
 #if os(iOS) || os(macOS)
 // MARK: Beams
-private let pushNotifications: PushNotifications = PushNotifications.shared
-
 extension ChatManager {
     /**
      Register device token with PushNotifications service.
 
      - Parameter deviceToken: A token that identifies the device to APNs.
      */
     public static func registerDeviceToken(_ deviceToken: Data) {
-        pushNotifications.registerDeviceToken(deviceToken)
+        PushNotifications.shared.registerDeviceToken(deviceToken)
```

Comment:
I thought we were not going to use the `shared` instance in Chatkit, so that it's available for customers in their own usage?

Reply:
That certainly appears to be how it was done in Android.

Reply:
You're correct that the intention is that one. On the swift SDK, this function will apply to all instances that are in a started state, so we can use the singleton here. Likewise for the other function, `registerForRemoteNotifications`, which just prompts the UI for the user to accept the push notifications permission.

Reply:
Shouldn't the functions be `static` then?

If not, then I think a comment is definitely in order.

Reply:
They'd both do the same thing. But I see your point, I've updated the PR to use `PushNotificationsStatic` instead of `PushNotifications.sahred` to make it clearer what's happening.

Reply:
`PushNotifications.shared` was left as a backwards compatibility, but it just points to the static version. But there's no reason why we can't be explicit here.